src/main/java/frc/robot/RobotContainer.java

```diff
@@ -64,8 +65,12 @@ private void configureBindings() {
     driverControllerBackButton.onTrue(new InstantCommand(drivetrainSubsystem::zeroGyroscope));
     operatorControllerXButton.onTrue(new FloorPickup(floorIntake));
     operatorControllerAButton.onTrue(new FloorDrop(floorIntake));
-    driverControllerYButton.onTrue(new ToggleRotationTarget(drivetrainSubsystem, () -> vision.getTargetXOffsetDegrees()));
     driverControllerBButton.onTrue(new ToggleRotationTarget(drivetrainSubsystem, () -> Constants.DRIVE_ROTATION_TARGET_DEGREES));
+    driverControllerXButton.toggleOnTrue(new AlignToTarget(vision, drivetrainSubsystem,
+    () -> mod.modifyAxis(driverController.getRawAxis(Constants.LEFT_X_AXIS))
+    * DrivetrainSubsystem.MAX_VELOCITY_METERS_PER_SECOND,
+    () -> -mod.modifyAxis(driverController.getRawAxis(Constants.LEFT_Y_AXIS))
+    * DrivetrainSubsystem.MAX_VELOCITY_METERS_PER_SECOND, false));
```

Comment:
Lets have two commands:

1. Set rotation target such that we are facing our driver station
2. Set rotation target such that we are facing our opponents driver station